model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -635,6 +635,61 @@ message StandardCoders {
   }
 }
 
+// Experimental: A representation of a Beam Schema.
+message Schema {
+  enum TypeName {
+    BYTE = 0;
+    INT16 = 1;
+    INT32 = 2;
+    INT64 = 3;
+    DECIMAL = 4;
+    FLOAT = 5;
+    DOUBLE = 6;
+    STRING = 7;
+    DATETIME = 8;
+    BOOLEAN = 9;
+    BYTES = 10;
+    ARRAY = 11;
+    MAP = 13;
```

Comment:
Notably, you don't need this. `FieldType` has a oneof and one of the alternatives can be an atomic type and the others are map, row, array, logical. I would recommend breaking out more types in this oneof, for example a BytesTypes we may likely want metadata indicating a coder. And we discussed - and I think should go ahead with - removing datetime as a primitive type.